license-maven-plugin/src/test/resources/styles/javapkg_style.txt

```diff
@@ -0,0 +1,16 @@
+
+/*-
```

Comment:
I just see the dash and your comment (https://github.com/mycila/license-maven-plugin/pull/109#issuecomment-240138871).

My concern is that in Java it is really not common to have this. 

Why the dash ? Which IDE reformat a header based on this "hook" ? I wonder if we shouldn;t have instead 2 different header style because `/*-` is not really common amongst Java devs and I am pretty sure people would like the standard one, which is just `/*`

Reply:
The `/*- .. */` is supposed to indicate that it shouldn't be reformatted:

http://www.oracle.com/technetwork/java/javase/documentation/codeconventions-141999.html#350